apps/web/src/pages/Skills/components/UpdateSkillShowcase.tsx

```diff
@@ -247,125 +261,67 @@ const UpdateSkillShowcase = ({
+                    disabled={isBusy || disabled}
+                    items={initialData.userSkills.map((userSkill) => ({
+                      id: userSkill.skill ?? "unknown",
+                      ...userSkill,
+                    }))}
+                    max={maxItems}
+                    add={
+                      <SkillBrowserDialog
+                        inLibrary={allUserSkills
+                          .map((userSkill) => userSkill.skill)
+                          .filter(
+                            (skill) =>
+                              !existingSkillsRankingFiltered.includes(skill.id),
+                          )}
+                        trigger={triggerProps}
+                        context="showcase"
+                        skills={allSkills.filter(
+                          (skill) =>
+                            !existingSkillsRankingFiltered.includes(skill.id),
+                        )}
+                        onSave={handleAdd}
+                        showCategory={false}
+                        noToast
+                      />
+                    }
+                    onItemsMoved={(items) =>
+                      handleUpdate({ userSkills: items })
+                    }
+                  >
+                    {initialData.userSkills.map((item, index) => (
+                      <SkillShowcaseCard
+                        key={item.skill}
+                        item={item}
+                        index={index}
+                        skills={allSkills}
+                        userSkillRanking={userSkillRanking}
                       />
-                      <Button
-                        type="button"
-                        mode="inline"
-                        color="secondary"
-                        onClick={() => {
-                          navigate(pageInfo.returnPath);
-                        }}
-                      >
-                        {intl.formatMessage(commonMessages.cancel)}
-                      </Button>
-                    </div>
-                  </form>
-                </FormProvider>
+                    ))}
+                  </CardRepeater.Root>
+                </div>
+                <Separator />
+                <div
+                  data-h2-display="base(flex)"
+                  data-h2-gap="base(x.5, x1)"
+                  data-h2-flex-wrap="base(wrap)"
+                  data-h2-flex-direction="base(column) l-tablet(row)"
+                  data-h2-align-items="base(flex-start) l-tablet(center)"
+                >
+                  <Button
+                    type="button"
+                    mode="inline"
+                    color="secondary"
+                    onClick={() => {
+                      navigate(pageInfo.returnPath);
+                    }}
+                  >
+                    {intl.formatMessage(commonMessages.return)}
+                  </Button>
```

Comment:
Can this be a link instead?

`<Link href={pageInfo.returnPath}>`

Reply:
Oh yeah, that's better.  I also promoted it to solid now that it's the primary action.
[switch to link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9592/commits/0f032ecad4d8e253916ffccb2e0892e1936eef21)